statements/2025-09-05 ActivityPub vs AT Protocol.md

```diff
@@ -0,0 +1,42 @@
+
+This statement is a call for cooling the temperature of discussions and a reminder to be respectful of each other and the huge amount of work everyone is putting in to build a better open social web. We do not win by tearing each other down, which only emboldens and empowers those who do not want either protocol to succeed. This statement was written following an initial discussion at this month's [Social Web Community Group meeting](https://www.w3.org/events/meetings/3cfaa5d7-5013-4d32-81be-6988e55fa9ef/20250905T130000/).
+
+### Co-signed by the following community members:
+
+* Emelia Smith (Author, Independent Fediverse / OAuth developer) — [@thisismissem@hachyderm.io](https://hachyderm.io/@thisismissem) / [@thisismissem.social](https://bsky.app/profile/thisismissem.social)
+* Dmitri Zagidulin (Social CG co-chair) -- [@dmitri@social.coop](https://social.coop/@dmitri)
+* Darius Kazemi (Social Web CG member) — [@darius@friend.camp](https://friend.camp/@darius)
+* Nick Gerakines (SmokeSignal + AT Proto Lexicon Community) [@ngerakines.me](https://bsky.app/profile/ngerakines.me)
+* Robert W. Gehl (author of Move Slowly and Build Bridges) [@rwg@aoir.social](https://aoir.social/@rwg)
+* Juliet Shen (Roost Tools, Robust Open Online Safety Tools) [@julietshen.bsky.social](https://bsky.app/profile/julietshen.bsky.social) / [julietshen@mstdn.social](https://mstdn.social/@julietshen)
+* Erlend Sogge Heggen (Roomy, Muni Town) [@erlend.sh](https://bsky.app/profile/erlend.sh) / [@erlend@writing.exchange](https://writing.exchange/@erlend)
+* Jesse Karmani (frequency) [@jesseplusplus@mastodon.social](https://mastodon.social/@jesseplusplus)
+* Tantek Çelik (former Social Web Working Group co-chair) [@tantek.com](https://tantek.com/)
+* Bailey Townsend (Independent AT Protocol Developer) [@baileytownsend.dev](https://bsky.app/profile/baileytownsend.dev)
+* Anuj Ahooja (A New Social Executive Director & CEO) [@quillmatiq.com](https://bsky.app/profile/quillmatiq.com) / [@quillmatiq@mastodon.social](https://mastodon.social/@quillmatiq)
+* Johannes Ernst (Dazzle Labs, FediForum) [@j12t@j12t.social](https://j12t.social/@j12t) [@j12t.org](https://j12t.org)
+* James (Former SocialCG co-chair) [jamesg.blog](https://jamesg.blog/) (@[jamesg.blog@jamesg.blog](https://fed.brid.gy/r/https://jamesg.blog/))
+* Chris Messina (Citizen Agency, creator of ActivityStreams) [@chrismessina.me](https://chrismessina.me/)
+* Tim Chambers, founder of Indieweb.social [@tchambers@indieweb.social](https://indieweb.social/@tchambers)
+* Bryan Newbold (AT Protocol, Bluesky Social) [@bnewbold.net](https://bsky.app/profile/bnewbold.net) [@bnewbold@social.coop](https://social.coop/@bnewbold)
+
```

Comment:
* David Buchanan (AT Protocol, Bluesky Social) [@retr0.id](https://bsky.app/profile/retr0.id) / `@retr0id@retr0.id`

(latter is intentionally not linkified, I don't have a public web frontend on my instance)